src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -69,6 +70,8 @@ web-ext has the following commands; options specific to these commands are inclu
 
 Packages an extension into a `.zip` file, ignoring files that are commonly unwanted in packages, such as `.git` and other artifacts. The name of the `.zip` file is taken from the [name](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/name) field in the extension [manifest](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json).
 
+<section id="as-needed"></section>
```

Comment:
Question: what happens when you wrap the whole section into the tag?

```suggestion
<section id="as-needed">

#### `--as-needed`

[...]

</section>
```

This would be a bit better (semantically speaking).

(also, I wish we'd fix https://github.com/mozilla/extension-workshop/issues/497 instead of introducing new sections but that's a lot more work)

Reply:
> Question: what happens when you wrap the whole section into the tag?

That works too, does the same thing from the linking perspective. I mainly chose here a way to match what already existed.

#497 would be my next step. I need to look into what the best way would be and how it's done in MDN. I agree it should work like that, this PR is mostly a quick-fix to get them linkable first. Didn't want to introduce that refactoring into this PR since I wasn't quite sure how long it'll take me to make that happen in a good way.

If I fix this PR to be along the lines of your suggested change and then I'll look into #497 as a separate issue next, would that be a good approach?

Reply:
Yep, sounds good

Reply:
Made the section changes in commit [282ca05](https://github.com/mozilla/extension-workshop/pull/1438/commits/282ca057168eeb08c1fb54f4731f1d610b24c318).